uberfire-extensions/uberfire-metadata/uberfire-metadata-backends/uberfire-metadata-backend-elasticsearch/src/main/java/org/uberfire/ext/metadata/backend/elastic/index/ElasticSearchIndexProvider.java

```diff
@@ -313,8 +313,16 @@ private String escape(String s) {
         return sb.toString();
     }
 
-    private List<String> indicesToLowerCase(List<String> indices) {
-        return indices.stream().map(String::toLowerCase).collect(Collectors.toList());
+    protected List<String> prepareIndices(List<String> indices) {
+        return indices.stream()
+                .map(index -> index.toLowerCase().replaceAll("/",
+                                                             "_"))
+                .collect(Collectors.toList());
+    }
+
+    protected String prepareIndex(String index) {
```

Comment:
This method name is a bit confusing, because `org.elasticsearch.client.Client` which you're using in this class already has a method like this which does something radically different.

Perhaps you could rename it to `sanitizeString`, `sanitizeIndexName`, `replaceInvalidChars` or something like that.